EIPS/eip-6404.md

```diff
@@ -238,16 +241,17 @@ class RlpBlobTransactionPayload(Profile[TransactionPayload]):
     value: uint256
     input_: ByteList[MAX_CALLDATA_SIZE]
     access_list: List[AccessTuple, MAX_ACCESS_LIST_SIZE]
-    max_priority_fees_per_gas: BlobFeesPerGas
+    max_priority_fees_per_gas: BasicFeesPerGas
```

Comment:
why is this basic fees? it carries both basic and blob fee

Reply:
Priority fee is for the execution, and that also carries the max fee per gas. Blob only has `maxFeePerBlobGas` and should thus not have an entry in the priority fees for blobs